crates/bevy_sprite/src/render/mod.rs

```diff
@@ -920,12 +920,22 @@ pub type DrawSprite = (
     DrawSpriteBatch,
 );
 
+/// This struct implements the `RenderCommand` trait to bind GPU light data to the render pipeline
+/// at the specified bind group index `I`.
+/// `I` - A compile-time constant that specifies the bind group index where the point light
+/// data should be bound in the render pipeline.
 pub struct SetPointLightBindGroup<const I: usize>;
 impl<const I: usize> RenderCommand<Transparent2d> for SetPointLightBindGroup<I> {
     type Param = SRes<GpuLights2D>;
     type ViewQuery = &'static ViewUniformOffset;
     type ItemQuery = ();
 
+    /// Executes the render command to bind the point light data.
+    ///  `_item` - The transparent 2D render item (unused in this implementation)
+    ///  `_view` - The view uniform offset data (unused in this implementation)
+    ///  `_item_query` - The result of the item query (unused since ItemQuery is `()`)
+    ///  `lights_resource` - The GPU lights resource containing the bind group with light data
+    ///  `pass` - The tracked render pass used to record GPU commands
```

Comment:
This is just a hunch since I don't have a way to check but does this get formatted as a list correctly?

If not I would suggest something like:
```
* `_item` - The transparent 2D render item (unused in this implementation)
* `_view` - The view uniform offset data (unused in this implementation)
...
```